apps/webapp/app/services/runs/getRun.server.ts

```diff
@@ -0,0 +1,70 @@
+import { PrismaClient, prisma } from "~/db.server";
+import { z } from "zod";
+
+const GetRunInputSchema = z.object({
+  runId: z.string(),
+  maxTasks: z.number().default(20),
+  taskDetails: z.boolean().default(false),
+  subTasks: z.boolean().default(false),
+  cursor: z.string().optional(),
+});
+
+export type GetRunInput = z.infer<typeof GetRunInputSchema>;
+
+export class GetRunService {
```

Comment:
We use "presenters" instead of "services" for encapsulating non-mutating logic. So this should be renamed `ApiRunPresenter` and moved to `/apps/webapp/app/presenters`